new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonTestUtils.java

```diff
@@ -1892,6 +1892,7 @@ public static void createPV(String pvName, String domainUid, String className) {
     logger.info("creating persistent volume for pvName {0}, domainUid: {1}, className: {2}",
         pvName, domainUid, className);
     Path pvHostPath = null;
+    // when tests are running in local box the PV directories needs to exist
```

Comment:
typo: "… the PV directories needs" -> "… the PV directories need".

Reply:
Fixed